### DISCUSSION TITLE
Clarification about releases

### DISCUSSION THREAD

**Discussion by courtland** [florianl/go-tc#157] [Q&A] [answered]
Hello, and thanks for this great project. Im wondering if someone can shed light on whether or not to expect releases/tags in the future? This [comment](https://github.com/florianl/go-tc/issues/13#issuecomment-656822442) indicates there will not be proper "tagged" releases in the future, but I see some git tags since that comment.

My team is looking to leverage [this commit](https://github.com/florianl/go-tc/commit/d427ba03aa1bf38eabfd65a42ac8c2e5274d4206)  in [tc_exporter](https://github.com/fbegyn/tc_exporter) and wondering how best to proceed with a PR to `tc_exporter`. Should we expect a release or just reference the commit hash?

Thank you!

**Comment by florianl** [comment, marked as answer]
Hi @courtland 
Thanks for your interest in this project!
For the moment I'm not planing regular release cycles. Instead increasing git tags will be used from time to time. I don't remember the exact problem from https://github.com/florianl/go-tc/issues/13#issuecomment-656822442.

There are now more than six months, since the tagging of [v0.4.3](https://github.com/florianl/go-tc/releases/tag/v0.4.3). So tagging a new version sounds good to me.

**Update**
I just tagged the last commit with [v0.4.4](https://github.com/florianl/go-tc/releases/tag/v0.4.4) - hope this helps :)

**Reply by courtland** [reply]
Thank you!
